posts/Yoann/2018-11-29-hibernate-search-6-0-0-Alpha1.adoc

```diff
@@ -0,0 +1,179 @@
+= Get a sneak peek into the future of Hibernate Search with 6.0.0.Alpha1!
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search", "Lucene", "Elasticsearch", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We just published Hibernate Search 6.0.0.Alpha1,
+the first release for the next major version of Hibernate Search.
+
+This version is **not suitable for production** due to its limitations and to the fact APIs are still unstable.
```

Comment:
That's absolutely right, but I'm not sure I want to say this in a blog post targeting users. It could send implicit messages that I don't want to send:

* "Search 5 is not maintainable": it is maintainable since we're maintaining it.
* "They're breaking everything just for the beauty of it": we're not, there are legitimate reasons to break APIs.
* "Search 6 is going to be blazing fast": we haven't run performance tests yet, there's no way to tell for sure.
* "Search 6 will be bring us new features at unprecedented pace": it depends first and foremost on how many people work on Search. So far it looks good, but I don't want to over-promise.

I think what I'm trying to say is: the redesign has great potential, but I'm not sure it's something we can use to "promise" anything... Let the results speak for themselves :)